src/main/java/iitema/gypsypokemon/elements/ProjectileInterface.java

```diff
@@ -4,18 +4,22 @@
  * A projectile which can be shot
  */
 public interface ProjectileInterface {
+    /**
+     * Color of projectile
+     */
+    enum CartridgeColor {YELLOW, BLUE}
```

Comment:
I know, but according to 'clean coding standards' we should write this out anyway for more concise and readable code :)